packages/@expo/config-plugins/src/ios/Target.ts

```diff
@@ -77,6 +88,9 @@ function getTargetDependencies(
     return {
       name: trimQuotes(target.name),
       type,
+      signable: signableTargetTypes.some((signableTargetType) =>
+        isTargetOfType(target, signableTargetType)
```

Comment:
I would feel more comfortable with reversed conditions, it would be safer to keep list of types that should not be signed and assume that any unknown types needs signing.
This way we would be sure that we are not introducing any regressions.